docs/concepts/infrastructure.md

```diff
@@ -381,11 +381,11 @@ Additionally, we have "convenience" tags which are updated with each release to
 
 For example, when release `2.1.1` is published:
 
-1. Images with the release packaged are built for each supported Python version (3.7, 3.8, 3.9, 3.10, 3.11) with both standard Python and Conda.
-2. These images are tagged with the full description, e.g. `prefect:2.1.1-python3.7` and `prefect:2.1.1-python3.7-conda`.
-3. For users that want more specific pins, these images are also tagged with the SHA of the git commit of the release, e.g. `sha-88a7ff17a3435ec33c95c0323b8f05d7b9f3f6d2-python3.7`
-4. For users that want to be on the latest `2.1.x` release, receiving patch updates, we update a tag without the patch version to this release, e.g. `prefect.2.1-python3.7`.
-5. For users that want to be on the latest `2.x.y` release, receiving minor version updates, we update a tag without the minor or patch version to this release, e.g. `prefect.2-python3.7`
+1. Images with the release packaged are built for each supported Python version (3.8, 3.9, 3.10, 3.11) with both standard Python and Conda.
```

Comment:
is this a typo?

Reply:
Yep. Probably quite old. Good catch.

Reply:
Or maybe it's not a typo. "Images with the release packaged" - I remember having that thought that it was then that it wasn't. I think it's ok.